src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/azure/AzureProvider.java

```diff
@@ -479,7 +478,7 @@ public void deleteInstance(Infrastructure infrastructure, String instanceId) {
         // Delete the virtual networks if not attached to any remaining network interface
         deleteNetworks(azureService, networks);
 
-        logger.info("Deletion of all Azure resources of instance " + instanceId + " has been executed.");
+        log.info("Deletion of all Azure resources of instance " + instanceId + " has been executed.");
     }
```

Comment:
Maybe you can do this?
`log.info("Deletion of all Azure resources of instance {} has been executed.", instanceId);`

Reply:
This is how is it done in all the code. I think I would go too far from the intial aim of my task if I refactor the whole code. Is it ok for you ?

Reply:
Or do you think that I should refactor each log ?

Reply:
it is fine, keep it like this.